### PR TITLE
Add

### PR DESCRIPTION
**Pull request by Smelky**

### CODE REVIEW

File: src/com/concurrent/queues/Create.java [hunk 1]
```diff
@@ -1,5 +1,26 @@
 package com.concurrent.queues;
 
-public class Create {
+import java.util.concurrent.BlockingQueue;
 
+public class Create implements Runnable {
```
Comment (Taniann) [inline src/com/concurrent/queues/Create.java]: Create what, please choose more meaningful class names

File: src/com/concurrent/queues/Used.java [hunk 2]
```diff
@@ -1,5 +1,25 @@
 package com.concurrent.queues;
 
-public class Used {
+import java.util.concurrent.BlockingQueue;
 
+public class Used implements Runnable {
+    private BlockingQueue queue;
+    private int count;
+
```
Comment (Taniann) [inline src/com/concurrent/queues/Used.java]: between fields and constructors should be one blank line

File: src/com/concurrent/singleton/Singleton.java [hunk 3]
```diff
@@ -1,5 +1,18 @@
 package com.concurrent.singleton;
 
 class Singleton {
```
Comment (Taniann) [inline src/com/concurrent/singleton/Singleton.java]: add acces modificator to class, better dont use default one

Comment (Taniann) [reply src/com/concurrent/singleton/Singleton.java]: and now this class Singleton you can create not just one instance
correct it

Comment (Smelky) [reply src/com/concurrent/singleton/Singleton.java]: did it

**Pull request merged**